docs/examples/defaultTools.html

```diff
@@ -138,122 +157,124 @@
-        // remove useless tools for the live example
-        tools = _.without(
-          tools,
-          "CustomMouseWheelScroll",
-          "StackScrollMouseWheel",
-          "TextMarker",
-          "ZoomTouchPinch",
-          "PanMultiTouch",
-          "OrientationMarkers",
-          "ScaleOverlay",
-          "ManualLengthPlot",
-          "WSToggle",
-          "Overlay"
-        );
-        // remove not working tools for the live example
-        // see open issue
-        tools = _.without(
-          tools,
-          "Brush",
-          "ThresholdsBrush",
-          "RectangleScissors",
-          "FreehandScissors",
-          "CircleScissors",
-          "CorrectionScissors",
-          "PolylineScissors"
-        );
-        let increment = e.shiftKey ? -1 : 1;
-        tool_counter =
-          tool_counter == tools.length - 1 ? 0 : tool_counter + increment;
-        tool_counter = tool_counter < 0 ? 0 : tool_counter;
-        let tool = tools[tool_counter];
-        larvitar.setToolActive(tool, { mouseButtonMask: 1 });
-        $("#active-tool").html("Active Tool: " + tool);
+              // NOTE: this also activate the tools marked as default for each mouse button
+              larvitar.addMouseKeyHandlers(mouseConfig);
+            });
+          })
+          .catch(err => console.log(err));
       }
-    };
-  </script>
-</body>
 
-</html>
\ No newline at end of file
+      let demoFileList = getDemoFileNames();
+      _.each(demoFileList, function (demoFile) {
+        createFile(demoFile, renderSerie);
+      });
+
+      let tool_counter = 0;
+      document.onkeypress = function (e) {
+        e = e || window.event;
+        if (e.keyCode == 116 || e.keyCode == 84) {
+          let tools = _.map(larvitar.DEFAULT_TOOLS, "name");
+          // remove useless tools for the live example
+          tools = _.without(
+            tools,
+            "CustomMouseWheelScroll",
+            "StackScrollMouseWheel",
+            "TextMarker",
+            "ZoomTouchPinch",
+            "PanMultiTouch",
+            "OrientationMarkers",
+            "ScaleOverlay",
+            "ManualLengthPlot",
+            "WSToggle",
+            "Overlay"
+          );
+          // remove not working tools for the live example
+          // see open issue
+          tools = _.without(
+            tools,
+            "Brush",
+            "ThresholdsBrush",
+            "RectangleScissors",
+            "FreehandScissors",
+            "CircleScissors",
+            "CorrectionScissors",
+            "PolylineScissors"
+          );
+          let increment = e.shiftKey ? -1 : 1;
+          tool_counter =
+            tool_counter == tools.length - 1 ? 0 : tool_counter + increment;
+          tool_counter = tool_counter < 0 ? 0 : tool_counter;
+          let tool = tools[tool_counter];
+          larvitar.setToolActive(tool, { mouseButtonMask: 1 });
+          $("#active-tool").html("Active Tool: " + tool);
```

Comment:
We can remove jquery since it's used only in this example just to change an html string 😅

Reply:
it is used and loaded only in the html using cdn, it is not part of the library. But anyway we can remove it from the example